charts/hivemq-platform-operator/values.yaml

```diff
@@ -34,8 +34,15 @@ https:
 # Configures the Java JVM runtime options for the process.
 javaOpts: "-XX:+UnlockExperimentalVMOptions -XX:InitialRAMPercentage=75 -XX:MaxRAMPercentage=75"
 
-# Selector name that is used to watch HiveMQ Platforms with a matching selector.
+# An optional list of comma-separated namespace names that will be managed by this Operator instance.
+namespaces: ""
+# Selector name to configure which HiveMQ Platform instances are managed by this operator.
+# Use "operator.selector" in the hivemq-platform Helm chart to configure a matching label.
+# Note: This will create a label selector "operator=<my-selector-value>" for the operator.
 selector: ""
+# An optional list of comma-separated label selectors that HiveMQ Platform resources must match to be managed by this Operator instance.
+# Use "operator.labels" in the hivemq-platform Helm chart to configure matching labels for these label selectors.
```

Comment:
I wanted to prevent that, because of the many Helm CLI issues that are related to that. I would rather encourage customers to do this in a `values.yaml` file.